src/web/src/components/BannerButtons.tsx

```diff
@@ -8,6 +8,7 @@ import PopUp from './PopUp';
 
 const useStyles = makeStyles((theme) => ({
   buttonsContainer: {
+    paddingTop: ' env(safe-area-inset-top, 0)',
```

Comment:
Looks like there is an extra space prior to `env(...`. Rather than that, looks good on my iPhone!

Reply:
Good catch, will fix.